src/main/java/uk/ac/cam/cl/dtg/util/ReflectionsUtil.java

```diff
@@ -0,0 +1,53 @@
+package uk.ac.cam.cl.dtg.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class ReflectionsUtil {
+  public static List<Class<?>> getClasses(String packageName) throws ClassNotFoundException, IOException,
```

Comment:
Another small but important one is to consider if `List` is the right type. Every time I need to use some sort of collection, I always stop to ask myself if the type is right. For instance, does the return type need to be ordered? Can its elements be repeated? Asking these questions is good because it will force you to think about the correct type that's being returned. I have the feeling that the right type here may be a `Set`, because I suspect that these classes should be unique.